jinahub/encoders/text/DPRTextEncoder/tests/unit/test_encoder.py

```diff
@@ -23,7 +23,7 @@ def basic_encoder_ctx() -> DPRTextEncoder:
 
 
 def test_config():
-    encoder = BaseExecutor.load_config('../../config.yml')
+    encoder = Executor.load_config('../../config.yml')
```

Comment:
This one should have `__file__` or `cur_dir` as reference

Reply:
Good catch, thanks